### PR TITLE
[+]  Users can enable search on Smart Collections with isSearchable field

### PR DESCRIPTION
**Pull request by pierreco**

### CODE REVIEW

**Comment by pierreco** [comment]
[(?) Spendesk want to see their search bar on a smart collection with the search implemented](https://trello.com/c/ka165Rd7/1486-spendesk-want-to-see-their-search-bar-on-a-smart-collection-with-the-search-implemented)

File: index.js [hunk 1]
```diff
@@ -282,6 +284,7 @@ exports.collection = function (name, opts) {
     // NOTICE: Smart Collection definition case
     opts.name = name;
     opts.isVirtual = true;
+    opts.isSearchable = opts.isSearchable ? true :  false;
```
Comment (arnaudbesnier) [inline index.js]: `opts.isSearchable = opts.isSearchable ? true : false;`

can be simplified into 

`opts.isSearchable = !!opts.isSearchable;`

File: index.js [hunk 2]
```diff
@@ -267,6 +267,8 @@ exports.collection = function (name, opts) {
     Schemas.schemas[name].actions = _.union(opts.actions, Schemas.schemas[name].actions);
     Schemas.schemas[name].segments = _.union(opts.segments, Schemas.schemas[name].segments);
 
+    opts.isSearchable = true;
```
Comment (arnaudbesnier) [inline index.js]: I don't understand what it does and in which case it does something.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Smart Collections - Users can enable search on Smart Collections
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: `- Smart Collections - Add a new isSearchable property to display the search bar for Smart Collections.`

Comment (pierreco) [inline CHANGELOG.md]: done

File: index.js (hunk 1 above)
Comment (pierreco) [inline index.js]: done

File: index.js (hunk 2 above)
Comment (pierreco) [inline index.js]: removed

**Pull request merged**